### ISSUE TITLE
UWP Not working with DateTime object

### ISSUE THREAD

**Issue by IeuanWalker** [edsnider/LocalNotificationsPlugin#23]
If this issue is for a new feature, please skip the "Bug" section and fill out the "Feature Request" section at the bottom.

## Bug
When using the plugin with without a DateTime object the notification work as expected, but with the DateTime object in UWP app I do not receive a notification

Plugin Version:  2.1.0
Platform (e.g., Android v21):  UWP
Device/Hardware:  Local machine and windows phone emulator

### Expected behavior
Recive a notification
### Actual behavior
Do not receive a notification
### Steps to reproduce the behavior
- Create a Xamarin.forms PCL solution
- On a button click, I have added the following code
- CrossLocalNotifications.Current.Show("Test Title", "Test Body", 111, DateTime.Now.AddSeconds(5));

## Feature Request

Feature description:

**Comment by mapgccv** [comment]
Same behaviour here, compiling for UWP version 14393.

**Comment by mapgccv** [comment]
Spotted the problem(s): in 
https://github.com/edsnider/LocalNotificationsPlugin/blob/013c4345d25cd5df2f9de38f1f5a72d06ddb733f/src/Plugin.LocalNotifications.WinRT/LocalNotificationsImplementation.cs#L64 and many other places, a Tile updater is created, which is then passed a toast notification structure.

**Comment by IeuanWalker** [comment]
@mapgccv Wouldn't the problem be in the **.UWP** version, not the **.WinRT**?

**Comment by mapgccv** [comment]
@IeuanWalker: fair point. I have never developed a plugin and I'm not sure how, in case of the different versions windows, the platform-specific implementations are resolved at runtime. But looking at all windows projects, this is the only implementation I see, the other projects are empty, so I guess this file has got to be used, somewhat.

Moreover, I have tried it locally, by doing the correction I mention (namely:
creating a `ScheduledToastNotification` in place of a `ScheduledTileNotification` and queue it through a `ToastNotificationManager` instead of a `TileNotificationManager` and it works as expected - although I have tried it by manually replicating the plugin structure inside my application). I may try pushing the changes myself when I'll have a bit of time to learn how to test plugins.

Hope it helps.

**Comment by IeuanWalker** [comment]
Hi, @mapgccv could you create a simple project on GitHub so I can see how you implemented it?

**Comment by mapgccv** [comment]
Hi @IeuanWalker , in my solution I have modified the interface to accept a custom object, so the current source code would not apply to you. But, in short, I have included the interface in my .NET standard (or PCL) library, then copied the platform specific implementation files to the corresponding project and, in windows, I have replaced the function ``Show`` with ``DateTime`` with:

```
public void Show(string title, string body, int id, DateTime notifyTime)
{
      var xmlData = string.Format(_TOAST_TEXT02_TEMPLATE, title, body);
      var xmlDoc = new XmlDocument();
      xmlDoc.LoadXml(xmlData);
      var correctedTime = notifyTime <= DateTime.Now
        ? DateTime.Now.AddMilliseconds(100)
        : notifyTime;
      var scheduledToastNotification = new ScheduledToastNotification(xmlDoc, correctedTime)
      {
        Id = id.ToString()
      };
      ToastNotificationManager.CreateToastNotifier().AddToSchedule(scheduledToastNotification);
}
```

Hope this helps. If I find the time to learn a thing or two on plugins, I will try to contribute to this (as I have added custom sounds to notifications too).

**Comment by IeuanWalker** [comment]
@mapgccv that's great thanks, if you do find the time to update this plugin could you let me know thanks :)
